src/styles/utils/_vars.scss

```diff
@@ -1 +1,8 @@
-$c-gray: #eee;
```

Comment:
This line of code has been removed, you might need it in your SCSS file, please double-check this.